### PR TITLE
Changes in ways to auth and manipulate tests

### PR DESCRIPTION
**Pull request by erickants**

### CODE REVIEW

File: src/Auth/ToConnect.php [hunk 1]
```diff
@@ -15,13 +15,24 @@
...
 
     public function __construct(Authentication $paggcertoAuth = null,
-                                $endpoint = Paggcerto::ACCOUNT_ENDPOINT_SANDBOX)
+                                $endpointEnvironment = Paggcerto::ENDPOINT_SANDBOX)
     {
         $this->paggcertoAuthentication = $paggcertoAuth;
+        $this->endpointEnvironment = $endpointEnvironment;
+    }
+
+    public function getEndpointEnvironment()
```
Comment (Lowpoc) [inline src/Auth/ToConnect.php]: getEndpoint()

File: src/Service/HolderAccountService.php [hunk 2]
```diff
@@ -341,19 +360,59 @@ public function setTransferPlanAnticipated($anticipated)
...
+        $this->data->mobile = $mobilePhone;
+
+        return $this;
+    }
+
+    /**
+     * @param $comercialName
+     * @return $this
+     */
+    public function setComercialName($comercialName)
```
Comment (Lowpoc) [inline src/Service/HolderAccountService.php]: Ingles

File: src/Service/HolderAccountService.php [hunk 3]
```diff
@@ -341,19 +360,59 @@ public function setTransferPlanAnticipated($anticipated)
...
      * @return mixed
      */
     public function createHolderAccount()
     {
-        return $this->httpRequest(self::SIGNUP_SELLER, Requests::GET);
+        return $this->httpRequest(self::SIGNUP_SELLER, Requests::POST, $this->data);
     }
 
+    /**
+     *
```
Comment (Lowpoc) [inline src/Service/HolderAccountService.php]: ?

File: src/Service/PaggcertoAccountApiService.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ * Time: 11:59
+ */
+
+namespace Paggcerto\Service;
+
+
+use Paggcerto\Paggcerto;
+use stdClass;
+
+class PaggcertoAccountApiService extends PaggcertoService
```
Comment (Lowpoc) [inline src/Service/PaggcertoAccountApiService.php]: Payment e Account tem 7 caracteres  , abrevie

File: src/Auth/ToConnect.php (hunk 1 above)
Comment (erickants) [reply src/Auth/ToConnect.php]: Endpoint de ambiente

**Pull request merged**